docs/cloud-tutorials/application-management.md

```diff
@@ -47,6 +49,18 @@ If you edit your application, run `dbos-cloud app deploy` again to apply the lat
 * You cannot change the database of a deployed application. You must delete and re-deploy the application.
 :::
 
+#### Configuring Application Deployment
+In `dbos-config.yaml`, you can configure your application deployment with these properties:
```

Comment:
I would add a little more detail on exactly what this does.

I would also make the recommendation to use a `build.sh` and put all the commands there so the setup is just `./build.sh`.  Doing it this way runs into incomprehensible bash escaping rules.

Finally, note that this is for pro users only.